Swift/fibonacci/fibonacci.swift

```diff
@@ -1,12 +1,23 @@
-func fibonacci(n: Int) -> Int {
-var a = 0
-var b = 1
-for _ in 0..<n {
-let temp = a
-a = b
-b = temp + b
-}
-return a
+func FibonacciSequence(n: Int) -> Int {
+	if n <= 1 {
+		return n
+	}
+
+	return FibonacciSequence(n: n-1) + FibonacciSequence(n: n-2)
 }
 
-print(fibonacci(n:7))
+let arguments = CommandLine.arguments
+
+var n = 10
```

Comment:
this looks unused

Reply:
Its used, it declares the variable `n` as an integer.